src/main/groovy/io/gatling/gradle/GatlingEnterpriseUploadTask.groovy

```diff
@@ -15,14 +15,8 @@ class GatlingEnterpriseUploadTask extends DefaultTask {
         def gatling = project.extensions.getByType(GatlingPluginExtension)
         RecoverEnterprisePluginException.handle(logger) {
             EnterprisePlugin enterprisePlugin = gatling.enterprise.initBatchEnterprisePlugin(project.version.toString(), logger)
-
-            String jsonPackageConfig = PackageConfiguration.loadToJson(project.projectDir)
             UUID packageUUID = gatling.enterprise.packageId
-            if (jsonPackageConfig != null) {
-                logger.lifecycle("Package configuration file detected, applying it.")
-                packageUUID =  enterprisePlugin.uploadPackageConfiguration(jsonPackageConfig)
```

Comment:
We need a milestone where `uploadPackageConfiguration` have been removed from commons